### PR TITLE
Understanding CSS Pseudo classes and Pseudo elements

### PR DESCRIPTION
**Pull request by ReactifyStudio**
### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [ ] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [ ] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [ ] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by ReactifyStudio** [comment]
This fixes #1253

**Comment by ReactifyStudio** [comment]
@kayere kindly reopen this PR, my last PR is merged. Regarding @hectorkambow request for one article at a time. 👍

**Comment by hectorkambow** [comment]
> @kayere kindly reopen this PR, my last PR is merged. Regarding @hectorkambow request for one article at a time. 👍

Lets wait for the payout to go through as that was the issue 👍  Thank you @ReactifyStudio

**Comment by ReactifyStudio** [comment]
Well 💯

On Feb 7, 2021 22:12, "hectorkambow" <notifications@github.com> wrote:

@kayere <https://github.com/kayere> kindly reopen this PR, my last PR is
merged. Regarding @hectorkambow <https://github.com/hectorkambow> request
for one article at a time. 👍

Lets wait for the payout to go through as that was the issue 👍 Thank you
@ReactifyStudio <https://github.com/ReactifyStudio>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/section-engineering-education/engineering-education/pull/1400#issuecomment-774739953>,
or unsubscribe

File: articles/css-pseudo-classes-elements/index.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+
+Cascading Style Sheet (CSS) is an independent language used to describe the presentation of web applications.
+As in any other language, one needs to understand the various concepts of the language.
+In this article, we will look at pseudo-classes and pseudo-elements concepts in CSS using practical examples.
+
+### Prerequisites
+This article focuses on the concepts of pseudo-classes and pseudo-elements. You will therefore need a basic understanding of other HTML and CSS concepts. Check the introduction to CSS and HTML topics.
```
Comment (kayere) [inline articles/css-pseudo-classes-elements/index.md:7]: Add links to the introduction to css and html topics.

File: articles/css-pseudo-classes-elements/index.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+### The CSS Pseudo Classes
+A pseudo-class is a CSS property that styles an element based on change to its state.
+A pseudo-class targets HTML elements that cannot be targeted by combinators or other selectors such us id and/or classes. This selector is used to define a special state of an element. An example is when a user hoovers the mouse over an element in a web page, or an input element gets a focus state.
+The syntax of pseudo-class looks like:
+```CSS
+selector:pseudo-class{
+    property:  value;
+}
+```
+Pseudo-classes are preceded by a colon (:) placed just after a CSS selector and then the name of the pseudo-class. It is important to note that the pseudo-classes are not case sensitive.
```
Comment (kayere) [inline articles/css-pseudo-classes-elements/index.md:18]: Restructure this sentence too.

**Review by kayere — CHANGES_REQUESTED** [review]
Hello @ReactifyStudio - I have done a preliminary review to your article and I'd like to make a suggestion before we move on. I have noticed that you have used the word 'will' in most parts. It doesn't give a good flow. I have made some changes but I'll leave the rest to you. Please go over the article to restructure sentences that use 'will'. Also make some general flow improvements.

**Comment by hectorkambow** [comment]
closing due to paypal issues - waiting for that to be cleared once again.

**Comment by ReactifyStudio** [comment]
Unless I use someone's account at the moment, I don't see solving this anytime soon. @hectorkambow just to complete the left PRs.

**Pull request closed without merging**